Controller/ProfilerController.php

```diff
@@ -117,4 +119,15 @@ private function explainOtherPlatform(Connection $connection, $query)
         return $connection->executeQuery('EXPLAIN ' . $query['sql'], $params, $query['types'])
             ->fetchAll(PDO::FETCH_ASSOC);
     }
+
+    /**
+     * @param mixed[] $query
+     */
+    private function explainOraclePlatform(Connection $connection, array $query)
+    {
+        $connection->executeQuery('EXPLAIN PLAN FOR ' . $query['sql']);
+
+        return $connection->executeQuery('SELECT * FROM TABLE(DBMS_XPLAN.DISPLAY)')
```

Comment:
- [Manual](https://docs.oracle.com/cd/B19306_01/server.102/b14211/ex_plan.htm#i26038) uses `DBMS_XPLAN.DISPLAY()` with brackets, not `DBMS_XPLAN.DISPLAY`. Why did you leave out brackets?
- Manual also uses `SELECT PLAN_TABLE_OUTPUT`, you do `SELECT *`, why?

Reply:
I have been following [DBMS_XPLAN](https://docs.oracle.com/database/121/ARPLS/d_xplan.htm#ARPLS70125), where `SELECT * FROM table(DBMS_XPLAN.DISPLAY)` is used. Parentheses can be omitted in Oracle if no parameter is given. I have no link to an official oracle documentation, but there is [some info about this on stackoverflow](https://stackoverflow.com/a/56511361). Also selecting all columns seems like the failsafe thing todo.

Reply:
So according that SO post it works without brackets by accident and it's not consistent across different functions. I would sleep better if we used documented version

Reply:
I have updated the PR to use `DBMS_XPLAN.DISPLAY()`, and fixed the missing `use 
 Doctrine\DBAL\Platforms\OraclePlatform`. To be fair to Oracle, it's inconsistent across different **types** of functions if parentheses are optional, mandatory or have to be omitted. It's not "by accident", or undocumented. See the docs for [User-Defined Functions](https://docs.oracle.com/cd/E11882_01/server.112/e41084/functions256.htm#SQLRF06181), [Single-Row Function SYSDATE](https://docs.oracle.com/cd/E11882_01/server.112/e41084/functions191.htm#SQLRF06124) or [Analytic Function ROW_NUMBER](https://docs.oracle.com/cd/E11882_01/server.112/e41084/functions156.htm#SQLRF06100)